trellis/core/node.hpp

```diff
@@ -87,6 +87,21 @@ class Node {
     return std::make_shared<PublisherClass<MSG_T>>(topic.c_str());
   }
 
+  /**
+   * CreateZeroCopyPublisher create a new handle for a zero-copy publisher
+   *
+   * @tparam MSG_T the message type that will be published by this handle
+   * @param topic the topic name to publish to
+   *
+   * @return a handle to a publisher instance
+   *
+   * Note: A zero-copy publisher should only be used for larger payloads (i.e. in the megabytes)
+   */
+  template <typename MSG_T>
+  Publisher<MSG_T> CreateZeroCopyPublisher(std::string topic) const {
```

Comment:
```suggestion
  Publisher<MSG_T> CreateZeroCopyPublisher(const std::string& topic) const {
```
A little ironic to have a "zero copy" function that copies the function arg :)

Reply:
Ha. You can rest assured knowing that string gets copied regardless... In fact the zero-copy implementation is not 100% zero copy either... Just on the receive side.